auth0/src/main/java/com/auth0/android/provider/OAuthManager.kt

```diff
@@ -55,7 +56,7 @@ internal class OAuthManager(
     }
 
     fun startAuthentication(context: Context, redirectUri: String, requestCode: Int) {
-        addRequiredScope(parameters)
+        OidcUtils.includeDefaultScope(parameters)
```

Comment:
moved the method to a different class, but it's the same logic (test remains unchanged)